jumpscale/data/serializers/blosc.py

```diff
@@ -0,0 +1,9 @@
+from blosc import compress, decompress
+
+
+def dumps(obj):
```

Comment:
See previous comment, do not follow the same naming for all serializes, here compress/decompress is the right one, you can just do as following:
```
import blosc

def compress(input):
    # first...convert input to byte-array if possible
   return blosc.comporess(input)
```

Reply:
in my code i gave you options to use compress,decompress,dumps and loads methods

Reply:
As the previous comment, dumps/loads is not the right naming.